### ISSUE TITLE
Diarization does not work

### ISSUE THREAD

**Issue by olsn** [JigsawStack/insanely-fast-whisper-api#4]
First of all: thank you for the nice project!
It works really well - except when, i try to activate diarization.

The docker-log then shows: 
`torchvision is not available - cannot save figures`

The response of the rest-call is:
```
{
  "detail": "'NoneType' object has no attribute 'eval'"
}
```

---

I have not yet tried to install torchvision, but maybe that's all that's neede?

**Comment by yoeven** [comment]
Yes torchvision is needed but it should be installed automatically if you're loading it from the dockerfile. Also you would need to provide a hugging face API key to load the diarization model. Check it out [here](https://github.com/JigsawStack/insanely-fast-whisper-api?tab=readme-ov-file#deploying-to-fly)
